ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -995,6 +1001,41 @@ - (void)observedScrollViewDidScroll:(UIScrollView *)scrollView {
     self.hairline.alpha = alpha;
 }
 
+- (NSArray *)stepsForReviewStep:(ORKReviewStep *)reviewStep {
+    NSMutableArray *steps = [[NSMutableArray alloc] init];
+    if (reviewStep.isStandalone) {
+        for (ORKStep *step in reviewStep.steps) {
+            [steps addObject:step];
+        }
+    } else {
+        ORKStep *nextStep = reviewStep;
```

Comment:
That way a review step could also act as an overview (tasks toc), which is an idea I like :-)